package.json

```diff
@@ -94,6 +92,7 @@
     "karma-coverage": "^0.5.3",
     "karma-firefox-launcher": "^0.1.6",
     "karma-jasmine": "^0.3.6",
+    "node-sass": "^3.4.1",
```

Comment:
@alexander-daniel can you bring `xml2js` down in the devDep too. 

It is only used in the proprecess step:

![image](https://cloud.githubusercontent.com/assets/6675409/12152753/3c711e28-b484-11e5-8b85-301076cf7035.png)